### ISSUE TITLE
Badge / banner for participating sites

### ISSUE THREAD

**Issue by WheresHJ** [codeforboston/internet_EBS#2]
We should enable IEBS participating sites to show their users that they are part of the IEBS network. To do this, we can provide:
- A static banner / badge declaring their participation
- A dynamic banner or badge that lists which cities this site will receive updates for in the event of an emergency

**Comment by jamiebuilds** [comment]
That may be to much information to fit into a small badge, it could always have a link to a page with more information. Plus an info page could explain more about the service and how other sites can get involved.

**Comment by WheresHJ** [comment]
Yup, definitely along the lines of what I was thinking for linking. In terms of information density, definitely _not_ thinking anything complex, just along the lines of "Supporting [cityname here]". A banner would more flexibility than a small badge, but it's all pretty fluid :)

Also just an idea, so not required for v1.

**Comment by jamiebuilds** [comment]
Websites are less likely to include a big banner. I think once something is built there are a lot of sites that will want to make posts about this.

**Comment by chrismetcalf** [comment]
How about an option where the script could place a little triangle banner
in the corner, like the "Fork me on Github" banner, but instead it'd be
small and just say "Internet EBS" or something like that. A mouseover could
list what cities that site is monitoring.

**Comment by jamiebuilds** [comment]
Again you're going to run into a lot more publishers that don't want that than ones that do.

**Comment by WheresHJ** [comment]
I think the concern about publishers is totally right on, which is why I was presenting a couple of ideas for display - and more options are good, including the option to not display anything. The underlying point is that a banner / badge / text block / whatever would allow sites that want to publicize their involvement with the program during non-emergency times to do so, and if they don't want to display it, it shouldn't be required.

It's more an attempt to allow participating sites to show their visitors that they are a place to go for information during an emergency, and to show that they are part of the support system and civic community in their city - not an attempt to enforce promotion of the IEBS product itself. We don't want to do anything that would block adoption, but I'm looking at this more like a business putting a "Cambridge Local First" sticker on their storefront window - if they're part of the program they _can_ put it in their window to tell people they're part of that community, but they don't have to by any means.

**Comment by beechnut** [comment]
I'm imagining an optional banner that looks good in a footer, in the vein of the Creative Commons banner: ![CC License](http://i.creativecommons.org/l/by-nc-sa/3.0/88x31.png)

Ours could be, perhaps: "[Internet EBS Logo]  |  NYC BOS SF", the abbreviations being whatever cities to which they subscribe.
